tests/Notifications/NotificationMailChannelTest.php

```diff
@@ -210,6 +210,20 @@ public function testMessageWithToAddress()
 
         $channel->send($notifiable, $notification);
     }
+
+    public function testMessageWithMailableContract()
+    {
+        $notification = new NotificationMailChannelTestNotificationWithMailableContract;
+        $notifiable = new NotificationMailChannelTestNotifiable;
+
+        $channel = new Illuminate\Notifications\Channels\MailChannel(
+            $mailer = Mockery::mock(Illuminate\Contracts\Mail\Mailer::class)
+        );
+
+        $mailer->shouldReceive('send')->once();
```

Comment:
Mailer would receive send anyway, I think you need to test that the Mailable object would receive send in this case.